src/pages/InfoPage.js

```diff
@@ -5,7 +5,7 @@ import styled from 'styled-components';
 const Container = styled.div`
   text-align: center;
   h2 {
```

Comment:
we should perhaps create an own components out of most of these styled component styles! :)